aries-site/src/examples/components/button/DosDonts/ButtonGoodCancelPreview.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Box, Button, Heading, Header, Text } from 'grommet';
+
+export const ButtonGoodCancelPreview = () => (
+  <Box width="large" gap="medium" pad="small">
+    <Header
+      direction="column"
+      align="start"
+      gap="xxsmall"
+      pad={{ horizontal: 'xxsmall' }}
+    >
+      <Heading level={2} margin="none">
+        Delete
```

Comment:
```suggestion
        Delete item?
```
From work done by Ephrem, there is recommendation to phrase the heading as a question that is specific about what will happen.

Reply:
got it thank you for that I will forward that info to keeth